README.md

```diff
@@ -131,7 +131,7 @@ Make sure you've read the [contribution guidelines](http://cakebuild.net/contrib
 
 ## License
 
-Copyright © .NET Foundation, Patrik Svensson, Mattias Karlsson, Gary Ewan Park, Alistair Chapman and contributors.
+Copyright © .NET Foundation, Patrik Svensson, Mattias Karlsson, Gary Ewan Park, Alistair Chapman, Martin Björkström, and contributors.
```

Comment:
@gep13 in these simple series I do find the comma unnecessary as removing it wouldn't lead to any misunderstandings. However, both works and are grammatically correct.

Before this change, there was no comma between the last name and contributors.

Does anyone else from @cake-build/cake-team have any thoughts on this?